poetry.lock

```diff
@@ -3570,32 +3570,32 @@ test = ["dask", "netcdf4", "pytest (>=3.6)", "pytest-cov", "pytest-timeout"]
 
 [[package]]
 name = "scipy"
-version = "1.9.2"
+version = "1.9.3"
```

Comment:
Not bumping to `scipy==1.10.0` or higher for now because they have set an upper pin on Python at https://github.com/scipy/scipy/blame/v1.10.0/pyproject.toml#L74. May eventually need to yield though because scipy now upper bounds Python - https://scipy.github.io/devdocs/dev/core-dev/index.html#updating-upper-bounds-of-dependencies